### PR TITLE
Check for license only in Cargo.toml

### PR DESCRIPTION
**Pull request by UK992**
This fix problem with false reporting about missing license in every toml file, that is not Cargo.toml

r? @Wafflespeanut 

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13027)

### CODE REVIEW

File: python/tidy/servo_tidy_tests/test_tidy.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def test_webidl(self):
         self.assertNoMoreErrors(errors)
 
     def test_toml(self):
-        errors = tidy.collect_errors_for_files(iterFile('test.toml'), [tidy.check_toml], [], print_text=False)
+        errors = tidy.collect_errors_for_files(iterFile('test-Cargo.toml'), [tidy.check_toml], [], print_text=False)
```
Comment (Wafflespeanut) [inline python/tidy/servo_tidy_tests/test_tidy.py]: Let the new name be lower-cased (like, `test-cargo.toml`)

File: python/tidy/servo_tidy/tidy.py [hunk 2]
```diff
@@ -312,7 +311,7 @@ def find_reverse_dependencies(dependency, version, content):
 
 
 def check_toml(file_name, lines):
-    if not file_name.endswith(".toml"):
+    if not file_name.endswith("Cargo.toml"):
```
Comment (Wafflespeanut) [inline python/tidy/servo_tidy/tidy.py]: Shouldn't this check be `file_name == 'Cargo.toml'` then?

Comment (UK992) [inline python/tidy/servo_tidy/tidy.py]: This won't work, because ``file_name`` is a path, not only file name.

Comment (Wafflespeanut) [inline python/tidy/servo_tidy/tidy.py]: Oh, right. nvm :)

**Comment by Wafflespeanut** [comment]
Thanks for the cleanup! :)

@bors-servo r+

File: python/tidy/servo_tidy_tests/test_tidy.py (hunk 1 above)
Comment (UK992) [inline python/tidy/servo_tidy_tests/test_tidy.py]: Renamed to ``Cargo.toml``

**Pull request merged**